src/parser/rogue/assassination/modules/talents/Nightstalker.tsx

```diff
@@ -52,11 +56,11 @@ class Nightstalker extends StealthCasts {
 
   get vanishCastsSpentOnRupture() {
     let vanishWithRupture = 0;
-    this.stealthSequences.forEach(sequence => {
+    this.stealthSequences.forEach((sequence: any) => {
       if (this.usedStealthOnPull && sequence === this.stealthSequences[0]) {
         return;
```

Comment:
Type instead of `any`

Reply:
This is a massive headache, might need to rewrite the logic for this and its pipeline